src/app/tabs/test-tab.js

```diff
@@ -482,11 +485,38 @@ module.exports = class TestTab extends ViewPlugin {
     return yo`<span class='text-info h6'>Progress: ${ready} finished (of ${this.runningTestsNumber})</span>`
   }
 
+  updateDirList (e) {
+    if (e.keyCode === 191) {
+      for (var o of this.uiPathList.querySelectorAll('option')) o.remove()
+      this.testTabLogic.dirList(this._view.el.getElementsByClassName('custom-select')[0].value).then((options) => {
```

Comment:
I think you should not fetch element from class name, doesn't look like robust..